src/components/Testimonial.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import clientAvatar from '../img/client.jpg';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import faQuoteLeft from '@fortawesome/fontawesome-free-solid/faQuoteLeft';
+
+export default ({
+  clientName = 'Juhi Jayaram',
```

Comment:
Just nitpicking, but prefer grouping relevant information in an object. Like `client`. And reference with `client.name` and `client.review`

Reply:
Yeah! Makes sense!